engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/instrument/RuntimeVisualizationsTest.scala

```diff
@@ -1351,6 +1351,17 @@ class RuntimeVisualizationsTest extends AnyFlatSpec with Matchers {
       )
     )
 
+    val attachVisualizationResponses =
```

Comment:
This test is [failing in CI](https://github.com/enso-org/enso/actions/runs/11057235037/job/30720561171#step:7:3916). It looks to me like it is a nondeterministic failure related to `sequentialExecution = false` command delays, and not related to this changeset. The observed failure mode is for all execution updates to show the result of executing the post-modification module. I think this would occur if delaying is applied to the `attachVisualization` commands, and not to the `textEdit` command.